logging_config.py

```
@@ -0,0 +1,497 @@
+        'request_id': request_id_var.get(),
+        'correlation_id': correlation_id_var.get()
+    }
+    
+    if context:
+        extra.update(context)
+    
+    logger.debug(message, extra=extra)
+
+
+# Initialize logging when module is imported
+import hashlib
```

Ruff (0.12.2)

Module level import not at top of file

(E402)